pkg/labels/matcher_test.go

```diff
@@ -82,8 +84,36 @@ func TestMatcher(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		if test.matcher.Matches(test.value) != test.match {
-			t.Fatalf("Unexpected match result for matcher %v and value %q; want %v, got %v", test.matcher, test.value, test.match, !test.match)
-		}
+		testutil.Assert(t, test.matcher.Matches(test.value) == test.match, "Unexpected match result for matcher %v and value %q; want %v, got %v", test.matcher, test.value, test.match, !test.match)
```

Comment:
My apologies, when I suggested `testutil.Assert`, I was actually thinking of `testutil.Equals`, where you automatically get standardized output for `want %v, got %v` type behavior. Would you be willing to update to use that function in these two tests?